README.md

```diff
@@ -4,6 +4,43 @@ react-native-fabric-crashlytics
 
 Reports javascript exceptions in React Native to the Crashlytics server, using the react-native-fabric library.
 
+Dependencies
+-----
+
+You need to install react-native-fs and link it to your main project to run this:
+
+```
+npm install -S react-native-fs
+react-native link react-native-fs
+```
+
+Setup iOS
```

Comment:
In theory, these could be done by a rpm-link too, but I get that it becomes more complex, and I'm not as familiar with how to do that, either. :)

At the very least, will your code work if these steps are *not* done?

Reply:
Without react-native-fs linked no. Unless the library is instantiated without the path as parameter, which falls back to the code without sourcemaps. Otherwise, how exactly would this work if there was no way to read the sourcemap?

Reply:
I think I'm asking "will this still post obfuscated errors if the dev doesn't do this, or is this a breaking change requiring a semver-major bump"?

Reply:
I might get a way around it but the person might be mislead and think that the actual lib is not working.

Reply:
So, the actual question is: should we fallback to the other code if the steps above are not done? Or should a better exception be thrown?

Reply:
I think I'm okay with making this a requirement, since it is cross-platform if you follow the instructions. I just need to make sure do a major-semver bump after this to avoid anyone accidentally including it.